packages/ui/tailwind-utils-config/components/multi-select.ts

```diff
@@ -56,6 +56,7 @@ export default {
     },
 
     '&-input': {
+      minWidth: '50px',
```

Comment:
why we are adding `minWidth` here?

Reply:
Without minWidth the input can be too small to type

<img width="675" height="146" alt="image" src="https://github.com/user-attachments/assets/754a3134-27bd-460b-99d7-9eb88e3ceb1b" />

Reply:
With this change it goes to the next line when there's not enough space